lib/i18n/tasks/translators/yandex_translator.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'i18n/tasks/translators/base_translator'
+
+module I18n::Tasks::Translators
+  class YandexTranslator < BaseTranslator
+    def initialize(*)
+      begin
+        require 'yandex-translator'
+      rescue LoadError
+        raise ::I18n::Tasks::CommandError, "Add gem 'yandex-translator' to your Gemfile to use this command"
+      end
+      super
+    end
+
+    protected
+
+    def translate_values(list, **options)
+      list.map { |item| translator.translate(item, options) }
+    end
+
+    def options_for_translate_values(from:, to:, **options)
+      options.merge(
+        from: to_yandex_compatible_locale(from),
+        to: to_yandex_compatible_locale(to)
+      )
+    end
+
+    def options_for_html
+      {}
```

Comment:
The Yandex API needs `format: 'html'` for translating HTML values
https://tech.yandex.com/translate/doc/dg/reference/translate-docpage/

I see that the yandex-translate gem doesn't support the `format` option yet.
We should make sure HTML keys are supported before merging this, because otherwise they will have broken markup.

Looks like there is an abandoned PR for this https://github.com/aegorov/yandex-translator/pull/13

Reply:
Opened a pr to do that work over there and updated the implementation here to utilize it:
https://github.com/aegorov/yandex-translator/pull/14